sources/packages/web/src/components/common/restriction/ViewRestriction.vue

```diff
@@ -35,39 +35,43 @@
                 " /></v-col
           ></v-row>
         </content-group>
-        <template
-          v-if="restrictionData.restrictionType !== RestrictionType.Federal"
-        >
-          <v-divider></v-divider>
-          <h3 class="category-header-medium mb-5">Resolution</h3>
-        </template>
-        <v-textarea
-          v-if="allowUserToEdit"
-          label="Resolution reason"
-          v-model="formModel.resolutionNote"
-          variant="outlined"
-          :rules="[(v) => checkResolutionNotesLength(v)]" />
-        <content-group
-          v-if="
-            !restrictionData.isActive &&
-            restrictionData.restrictionType !== RestrictionType.Federal
-          "
-        >
-          <title-value
-            propertyTitle="Resolution reason"
-            :propertyValue="restrictionData.resolutionNote"
+        <template v-if="canResolveRestriction">
+          <template
+            v-if="restrictionData.restrictionType !== RestrictionType.Federal"
+          >
+            <v-divider></v-divider>
+            <h3 class="category-header-medium mb-5">Resolution</h3>
+          </template>
+          <v-textarea
+            v-if="allowUserToEdit"
+            label="Resolution reason"
+            v-model="formModel.resolutionNote"
+            variant="outlined"
+            :rules="[(v) => checkResolutionNotesLength(v)]"
           />
-          <v-row
-            ><v-col
-              ><title-value
-                propertyTitle="Date resolved"
-                :propertyValue="restrictionData.updatedAt" /></v-col
-            ><v-col
-              ><title-value
-                propertyTitle="Resolved by"
-                :propertyValue="restrictionData.updatedBy" /></v-col
-          ></v-row> </content-group
-      ></template>
+          <content-group
+            v-if="
+              !restrictionData.isActive &&
+              restrictionData.restrictionType !== RestrictionType.Federal
+            "
+          >
+            <title-value
+              propertyTitle="Resolution reason"
+              :propertyValue="restrictionData.resolutionNote"
+            />
+            <v-row
+              ><v-col
+                ><title-value
+                  propertyTitle="Date resolved"
```

Comment:
THis conversion can be done here which will avoid the following error being shown.

![image](https://user-images.githubusercontent.com/54600590/236532742-01e1b548-616d-4dde-97a7-25229be7f203.png)

Error otherwise.

![image](https://user-images.githubusercontent.com/54600590/236532547-da35e068-1b30-4115-b3e2-d2d1ab309208.png)

Reply:
I see that code is not updated for this comment.

Reply:
I had made this change in the 1st revision of changes. I can see this change in github.

Reply:
As discussed there are other lines where we need to change. 
![image](https://github.com/bcgov/SIMS/assets/54600590/8c5f5732-652a-478e-bd28-48c9c940fe6b)

Reply:
As discussed, leaving `Status`, I have made the change.